HousingRepairsOnlineApi.Tests/GatewaysTests/AppointmentGatewayTests.cs

```diff
@@ -102,16 +103,18 @@ public async Task GivenValidParameters_WhenBookingAppointment_NoExceptionIsThrow
             const string SorCode = "SOR Code";
             const string LocationId = "Location ID";
             const string BookingReference = "Booking Reference";
+            const string RepairDescriptionText = "Repair description text";
             var startDateTime = new DateTime(2022, 01, 01, 8, 0, 0);
             var endDateTime = new DateTime(2022, 01, 01, 12, 0, 0);
-
-            mockHttp.Expect($"/Appointments/BookAppointment?bookingReference={BookingReference}&sorCode={SorCode}&locationId={LocationId}&startDateTime={startDateTime}&endDateTime={endDateTime}")
+            var jsonContent = JsonContent.Create(new { RepairDescriptionText });
```

Comment:
jsonContent is not used. Did you intend to use it in the BookAppointment function call below or is it left in from previous tests?